pynautobot/models/dcim.py

```diff
@@ -81,9 +80,6 @@ class Devices(Record):
 
     has_details = True
     device_type = DeviceTypes
-    primary_ip = IpAddresses
-    primary_ip4 = IpAddresses
-    primary_ip6 = IpAddresses
```

Comment:
Just to make sure I understand correctly - this is because the only thing that distinguished `IpAddresses` from the parent `Record` class was the custom `__str__` method, which is no longer needed, and therefore there's no longer any need to even have an `IpAddresses` class?

Reply:
what does the `display` attribute represent, and what scenarios do we expect it not to exist?

Reply:
> what does the `display` attribute represent, and what scenarios do we expect it not to exist?

In theory all model serializers in Nautobot should provide a `display` attribute that provides a relevant string representation of the object. In practice, it looks like some existing serializers have a gap here - see https://github.com/nautobot/nautobot/pull/1783#discussion_r875896753 for a list

Reply:
Sure, see https://github.com/nautobot/pynautobot/blob/develop/pynautobot/core/response.py#L212 - could re-add the fallback logic that was removed in #42 and have it try `name` and `label` if there's no `display`...?

Reply:
ya, I wasn't really sure if that was still appropriate, but if that makes sense, then I'm good with falling back to what was previously there

Reply:
I added it back within the latest commit so it'll check for **display** -> **name** -> **label** and then default to empty string.